tsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
+  "include": ["src/**/*"],
+  "compilerOptions": {
+    "allowJs": true,
```

Comment:
add "strict: true", please

Reply:
Done. 

Maybe it would be nice to have a recommended tsconfig and vite file for plugins.